library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
Minor nit, instead of these living in <repo root>/library and being symlinked into the role, they should simply live in the role.